Eslint error when run on files using React

**Describe the bug**
Eslint has not been configured to use react

    /Users/canhnd/achromevement/src/options/main.js
    5:8  error  'React' is defined but never used  no-unused-vars
    8:7  error  'App' is defined but never used    no-unused-vars

    /Users/canhnd/achromevement/src/popup/main.js
    6:8  error  'React' is defined but never used  no-unused-vars
    9:7  error  'App' is defined but never used    no-unused-vars

**To Reproduce**
Run: `npm run lint`

**Expected behavior**
No error

--- a/.eslintrc.json
+++ b/.eslintrc.json
@@ -3,12 +3,17 @@
   "rules": {
     "comma-dangle": [2, "always-multiline"],
     "semi": [2, "always"],
-    "standard/no-callback-literal": "off"
+    "standard/no-callback-literal": "off",
+    "react/jsx-uses-react": "error",
+    "react/jsx-uses-vars": "error"
   },
   "globals": {
     "describe": true,
     "test": true,
     "expect": true,
     "chrome": true
-  }
+  },
+  "plugins": [
+    "react"
+  ]
 }

--- a/package.json
+++ b/package.json
@@ -38,6 +38,7 @@
     "eslint-plugin-import": "^2.14.0",
     "eslint-plugin-node": "^7.0.1",
     "eslint-plugin-promise": "^4.0.1",
+    "eslint-plugin-react": "^7.11.1",
     "eslint-plugin-standard": "^4.0.0",
     "fancy-log": "^1.3.2",
     "gulp": "^4.0.0",

